### PR TITLE
My First Pull Request

### PR DESCRIPTION
**Pull request by zhadier**
**My First Pull Request**

In this Branch we
- Added Html file with Heading
- Added CSS file with color change for heading
- Added Linters and .gitignore
- Ignored test.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot](./app_screenshot.png)
+
```
Comment (rotimiazeez) [inline README.md]: - [ ] The screenshot link is broken Kindly fix it😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @zhadier 👍, 

### Good job so far!😉
### Summary!🎉
- No linter error✔
- Good PR message✔
- Descriptive README✔
- Good commit messages
#### There are some issues that you still need to work on to go to the next project but you are almost there!💯
> Just a few changes and you'd be unstoppable🚀😎

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (zhadier) [reply README.md]: It has been fixed. Thank you.

**Review by rotimiazeez — APPROVED** [review]
# Hi @zhadier , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**